test/functional/pdo_sqlsrv/pdo_getAttribute_clientInfo.phpt

```diff
@@ -1,16 +1,21 @@
 --TEST--
-Test client info by calling PDO::getAttribute with PDO::ATTR_CLIENT_VERSION 
+Test client info by calling PDO::getAttribute with PDO::ATTR_CLIENT_VERSION
+--SKIPIF--
+<?php require('skipif_mid-refactor.inc'); ?>
 --FILE--
```

Comment:
maybe use a regular skipif? or any other tests that don't involve creating table or inserting data

Reply:
I'll leave it since we may want to test the combination of a column encryption enabled connection and calling CLIENT_VERSION